contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -322,14 +330,13 @@ contract LoanTokenLogicStandard is LoanTokenSettingsLowerAdmin {
 		uint256 _newBalance,
 		uint256 _currentPrice
 	) internal {
-		// keccak256("iToken_ProfitSoFar")
-		bytes32 slot = keccak256(abi.encodePacked(_user, uint256(0x37aa2b7d583612f016e4a4de4292cb015139b3d7762663d06a53964912ea2fb6)));
+		bytes32 slot = keccak256(abi.encodePacked(_user, iToken_ProfitSoFar));
```

Comment:
check out [eip-1967](https://eips.ethereum.org/EIPS/eip-1967) approach to reduce collision chance even further

Reply:
@tjcloa did you make sure that this does not change the storage slot currently used by the contracts? (just want to be safe here)

Reply:
double checked once again just in case, created test contract - the result is identical